Source/Core/Core/NetPlayClient.cpp

```diff
@@ -484,11 +499,13 @@ void NetPlayClient::ThreadFunc()
 	{
 		ENetEvent netEvent;
 		int net;
+		if (m_traversal_client)
+			m_traversal_client->HandleResends();
+		net = enet_host_service(m_client, &netEvent, 4);
+		while (!m_async_queue.Empty())
```

Comment:
what is the point of using this queue rather than just sending as usual and interrupting enet_host_service()?